<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KategoriEkrani</class>
 <widget class="QWidget" name="KategoriEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kategori Ekranı</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>1340</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>1220</y>
     <width>851</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Bilgilerini görmek istediğiniz ürünün adı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>681</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>35</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>KATEGORİLER:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>1860</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>461</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_kategori">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>Seçiniz..</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Giyim</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Temizlik</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hediyelik Eşya</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Doğal Ürünler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diğer</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1560</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Devam</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
